tests/test_ignore_workbook_corruption_error.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+from xlrd.compdoc import CompDocError
+
+import xlrd
+from .base import from_this_dir
+
+
+class TestIgnoreWorkbookCorruption(TestCase):
+
+    def test_not_corrupted(self):
+        with self.assertRaises(Exception) as context:
+            xlrd.open_workbook(from_this_dir('corrupted_error.xls'))
+        self.assertTrue('Workbook corruption' in str(context.exception))
+
+        try:
+            xlrd.open_workbook(from_this_dir('corrupted_error.xls'), ignore_workbook_corruption_error=True)
+        except CompDocError:
+            self.fail('ignore_workbook_corruption_error=True, but the exception is raised.')
```

Comment:
I wouldn't bother with the try/except here, the original exception will be more useful.

Reply:
Did you mean just leave `xlrd.open_workbook(from_this_dir('corrupted_error.xls'), ignore_workbook_corruption_error=True)`?

Reply:
> I wouldn't bother with the try/except here, the original exception will be more useful.

Did you mean just leave` xlrd.open_workbook(from_this_dir('corrupted_error.xls'), ignore_workbook_corruption_error=True)`?